stress/n_workers_reputers__stress_test.go

```diff
@@ -272,16 +273,183 @@ func InsertReputerBulk(m TestMetadata,
+		if err != nil {
+			fmt.Println("Error creating funder address: ", topicFunderAccountName, " - ", err)
+			continue
+		}
+		topicFunderAddresses = append(topicFunderAddresses, topicFunderAddress)
+	}
+
+	err := fundAccounts(m, m.n.FaucetAcc, m.n.FaucetAddr, topicFunderAddresses, 1e9)
+	if err != nil {
+		fmt.Println("Error funding funder accounts: ", err)
+	} else {
+		fmt.Println("Funded", len(topicFunderAddresses), "funder accounts.")
+	}
+
+	getTopicFunder := func() (string, cosmosaccount.Account) {
+		topicFunderAccountName := getTopicFunderAccountName(topicFunderCount)
+		topicFunderCount++
+		topicFunderAccount, err := m.n.Client.AccountRegistry.GetByName(topicFunderAccountName)
+		if err != nil {
+			fmt.Println("Error getting funder account: ", topicFunderAccountName, " - ", err)
+			return "", topicFunderAccount
+		}
+		topicFunderAddress, err := topicFunderAccount.Address(params.HumanCoinUnit)
+		if err != nil {
+			fmt.Println("Error getting funder address: ", topicFunderAccountName, " - ", err)
+			return topicFunderAddress, topicFunderAccount
+		}
+		return topicFunderAddress, topicFunderAccount
+	}
+
+	workerCount := 1
+	reputerCount := 1
+
+	if topicsPerEpoch == 0 {
+		topicFunderAddress, topicFunderAccount := getTopicFunder()
+		WorkerReputerLoop(m, topicFunderAddress, topicFunderAccount, workerCount, reputerCount)
+		topicCount++
+	} else {
+		for {
+			startIteration := time.Now()
+
+			for j := 0; j < topicsPerEpoch && topicCount < topicsMax; j++ {
+				topicFunderAddress, topicFunderAccount := getTopicFunder()
+				go WorkerReputerLoop(m, topicFunderAddress, topicFunderAccount, workerCount, reputerCount)
+				topicCount++
+			}
+
+			workerCount += workersPerEpoch
+			reputerCount += reputersPerEpoch
+
+			elapsedIteration := time.Since(startIteration)
+			sleepingTimeSeconds := iterationTimeSeconds - elapsedIteration
+			for sleepingTimeSeconds < 0 {
+				sleepingTimeSeconds += iterationTimeSeconds
+			}
+			fmt.Println(time.Now(), "Main loop sleeping", sleepingTimeSeconds)
+			time.Sleep(sleepingTimeSeconds)
+		}
+	}
+}
+
+func getTopicFunderAccountName(topicFunderIndex int) string {
+	return "topicFunder" + strconv.Itoa(int(topicFunderIndex))
+}
+
+func getWorkerAccountName(workerIndex int, topicId uint64) string {
+	return "stressWorker" + strconv.Itoa(workerIndex) + "_topic" + strconv.Itoa(int(topicId))
+}
+
+func getReputerAccountName(reputerIndex int, topicId uint64) string {
+	return "stressReputer" + strconv.Itoa(reputerIndex) + "_topic" + strconv.Itoa(int(topicId))
+}
+
+// Register two actors and check their registrations went through
+func WorkerReputerLoop(
+	m TestMetadata,
+	topicFunderAddress string,
+	topicFunderAccount cosmosaccount.Account,
+	initialWorkerCount int,
+	initialReputerCount int,
+) {
+	topicId := SetupTopic(m, topicFunderAddress, topicFunderAccount)
+
+	report := func(a ...any) {
+		fmt.Println("[ TOPIC", topicId, "] ", a)
+	}
+
+	reputersPerEpoch := lookupEnvInt(m, "REPUTERS_PER_EPOCH", 0)
```

Comment:
Env vars are read in two places. 
I think they should be read outside (in the stress_test.go for example) and passed in as function parameters. This helps declaring all function dependencies.

Reply:
read separately in merged pr